src/Core/Common/UseParallelizeImageRegion/Documentation.rst

```diff
@@ -0,0 +1,32 @@
+Duplicate An Image
+==================
+
+.. index::
+   single: MultiThreaderBase
+
+Synopsis
+--------
+
+
+This example demonstrates how to take advantage of MultiThreaderBase::ParallelizeImageRegion.
+A comparison is made with LogImageFilter for purpose of computing log(1+x), where x is pixel value.
```

Comment:
We could provide more information here on why this is important, e.g.,

> With ``ParallelizeImageRegion``, we can process an image efficiently and in parallel with a function that is defined in a flexible way.  The entire operation, defined in 10 lines of code, previously required the definition of an entire class.

Reply:
Done